FlagsmithClient/src/main/java/com/flagsmith/entities/Trait.kt

```diff
@@ -6,11 +6,42 @@ import com.google.gson.annotations.SerializedName
 data class Trait(
     val identifier: String? = null,
     @SerializedName(value = "trait_key") val key: String,
-    @SerializedName(value = "trait_value") val value: String
-)
+    @SerializedName(value = "trait_value") val traitValue: Any
+) {
+
+    @Deprecated("Use traitValue instead or one of the type-safe getters", ReplaceWith("traitValue"))
+    val value: String?
+        get() = stringValue
```

Comment:
So as not to have to major version the client, can we add a setter and/or a constructor that accepts `value` still as well?

Reply:
The only difference for a user of the SDK is that value went from `String` to `String?` and carries with it the deprecation warning.

Reply:
Maybe we're better with the constructors then? I did have them in but took them out again as they seemed a bit OTT.

Reply:
This should be all sorted and tested now @matthewelwell